middlewares/isAuthorized.js

```diff
@@ -0,0 +1,28 @@
+import Listing from "../schemas/listingSchema.js";
+import Review from "../schemas/reviewSchema.js";
+
+const isListingAuthor = async (req, res, next) => {
+    let { id } = req.params;
+    let listing = await Listing.findById(id);
+
+    //Checking if user is actual owner of listing or not
+    if (!listing.owner.equals(res.locals.currUser._id)) {
+        req.flash("error", "Only owners can update the listing 🙅🏻‍♀️");
+        return res.redirect(`/listings/${id}`);
+    }
+    next();
+}
+
+const isReviewAuthor = async (req, res, next) => {
+    let { id, reviewId } = req.params;
+    let review = await Review.findById(reviewId);
+
+    //Checking if user is actual author of review or not
```

Comment:
always have a space after the // eg. `// Checking if user is actual author of review or not`